src/pages/components/button/usage.mdx

```diff
@@ -221,17 +221,18 @@ size.
-| _Field_      | 40/2.5          | Use when buttons are paired with input fields.                                                                                                                                                                                                                                     |
-| _Small_      | 32/2            | Use when there is not enough vertical space for the default or field-sized button.                                                                                                                                                                                                 |
+| Name               | Height (px/rem) | Use Case                                                                                                                                                                                                                                                                           |
+| ------------------ | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Full bleed_       | 64/4            | Use when buttons bleed to the edge of a larger component, like side panels or modals. _Note: This variant is not offered in the kit because it does not have a coded counterpart. Carbon only offers this button in the context of the [modal component](components/modal/usage)._ |
+| _Default_          | 48/3            | Use as primary page actions and other standalone actions.                                                                                                                                                                                                                          |
+| _Field_            | 40/2.5          | Use when buttons are paired with input fields.                                                                                                                                                                                                                                     |
+| _Small_            | 32/2            | Use when there is not enough vertical space for the default or field-sized button.                                                                                                                                                                                                 |
+| _Large expressive_ | 48/3            | Use when the body copy is 16px, or larger, such as in a marketing experience's lead space. This button works well with the expressive type set.                                                                                                                                    |
```

Comment:
We're not adding an additional size to the button since the default size is the same size used for expressive, so this can be removed.